ln-visualizer/docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: $APP_LN_VISUALIZER_WEB_IP
+      APP_PORT: 80
+
+  web:
+    image: maxkotlan/ln-visualizer-web:v0.0.20@sha256:9e4fff7600666ce27d1c835fa66e4a3fb34aabd7018506d749134baf9e8bc481
+    restart: on-failure
+    stop_grace_period: 1m
+    environment:
+      LN_VISUALIZER_API_URL: "http://${APP_LN_VISUALIZER_API_IP}:5647"
+    networks:
+      default:
+        ipv4_address: "${APP_LN_VISUALIZER_WEB_IP}"
+
+  api:
+    image: maxkotlan/ln-visualizer-api:v0.0.20@sha256:cfea255f3343dbacf96c5470dcd442f1cc034337dfb46ecfbbc6d5c1978da1a4
+    restart: on-failure
+    stop_grace_period: 1m
+    user: 1000:1000
```

Comment:
Also just wanted to mention that I just added this user line I saw in other umbrel apps.
I can't add it to the web right now because the web needs to override an nginx config, but figured it made sense to add to the api.

Reply:
@kroese oh it does? I was thinking it did the opposite haha. Thought it ran as a lower privileged user or something. Guess I'll be removing that...

Reply:
Ah okay. I think I will keep it for now then. Appreciate you checking.